shell/common/switches.h

```diff
@@ -273,6 +273,12 @@ DEF_SWITCH(
     "Setting this value to 0 or 1 disables MSAA. If it is not 0 or 1, it must "
     "be one of 2, 4, 8, or 16. However, if the GPU does not support the "
     "requested sampling value, MSAA will be disabled.")
+DEF_SWITCH(EnableIOSEmbedderAPI,
+           "enable-embedder-api",
+           "Enable the embedder api. Defaults to false. Ignored if the "
```

Comment:
Alternatively you could say "iOS only" for now, up to you though.

Reply:
My first version was actually "iOS only" but wanted to keep it future proof for android. I will revert back to "iOS only", it's less confusing.